.travis.yml

```diff
@@ -29,7 +29,7 @@ matrix:
       rvm: 2.4.4
     -
       env: PUPPET_GEM_VERSION="~> 4.0" CHECK=parallel_spec
-      rvm: 2.1.9
+      rvm: 2.3.1
```

Comment:
puppet4 is shipping with ruby 2.1.9, so we need to test against that ruby version

Reply:
reverted